src/form-fields/DatePicker.component.js

```diff
@@ -61,6 +68,17 @@ DatePicker.propTypes = {
     onChange: PropTypes.func.isRequired,
     dateFormat: PropTypes.string.isRequired,
     allowFuture: PropTypes.bool.isRequired,
```

Comment:
Also the `dateFormat` prop only has 2 valid values: `'dd-MM-yyyy'` and `'yyyy-MM-dd'`. It would be better to use `PropTypes.oneOf([ ... ])` validation.